server/lib/tuist/ci/job_run.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Tuist.CI.JobRun do
+  @moduledoc """
+  A job run represents execution of a CI job within a workflow.
+  This is a ClickHouse entity that stores CI job run data.
+  """
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @derive {
+    Flop.Schema,
+    filterable: [
+      :project_id,
+      :workflow_id,
+      :workflow_name,
+      :job_id,
+      :job_name,
+      :git_branch,
+      :git_commit_sha,
+      :runner_machine,
```

Comment:
@cschmatzler / @fortmarek should we have an id here that connects the job run with a machine?